zeeguu/api/utils/route_wrappers.py

```diff
@@ -1,11 +1,16 @@
 import functools
 import flask
 from zeeguu.logging import log
-from zeeguu.core.model.session import Session, User
+from zeeguu.core.model.session import Session
+
+from datetime import datetime, timedelta
 import zeeguu
 
+SESSION_CACHE = {}
+SESSION_CACHE_EXPIRE = 60  # Seconds
+
 
-def with_session(view):
+def has_session(view):
```

Comment:
i see the wrapper as an annotation for a method. 

take for example @classmethod. It is something that we want to say *about the method itself*. 

Thus, ours could say something like:  
E.g. 

```python
@requires_user_session
def modify_user_language():
```
or `requires_logged_in` maybe? or simply `requires_session`? or `expects_session`?  

does this make sense to you?

Reply:
Yes, let's go with `requires_session` guess that's quite self descriptive.